backend/api/Areas/Property/Models/Parcel/ParcelModel.cs

```diff
@@ -22,6 +22,16 @@ public class ParcelModel : PropertyModel
         public IEnumerable<ParcelFiscalModel> Fiscals { get; set; } = new List<ParcelFiscalModel>();
 
         public IEnumerable<ParcelBuildingModel> Buildings { get; set; } = new List<ParcelBuildingModel>();
+
+        /**
+         * All of the parcels related to this subdivision
+         */
+        public IEnumerable<SubdivisionParcelModel> Parcels { get; set; } = new List<SubdivisionParcelModel>();
```

Comment:
in other words, a subdivision parcel will only populate the Parcels array, and a Parcel with subdivisions will only populate the Subdivisions array.

Reply:
Is this more intuitive than having the property type provide the type?

Reply:
I hadn't really considered adding the property type here since that would allow for parcels to be given building designations and buildings to be given parcel designations. It would work, although it would continue our not great trend of enforcing our data model in logic and not in the database. If someone forgets to add/update the property type on these objects it would lead to invalid data, same with forgetting to update the project number array.

Reply:
You theoretically can still enforce this behaviour in the DB, but I won't recommend triggers.  It's easier to manage in the DAL.  The more explicit implementation is different types `ParcelType` and `BuildingType`

Reply:
Although this method requires that you correctly manage the type by inserting your data into the correct array, so it isn't perfect either.